--- /dev/null
+++ b/CMakeLists.txt
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 4.4.0)
+
+project(qlibs_ut
+  LANGUAGES CXX
+)
+
+add_library(qlibs.ut OBJECT)
+add_library(qlibs.ut::ut ALIAS qlibs.ut)
+
+target_sources(qlibs.ut
+  PUBLIC
+    FILE_SET CXX_MODULES
+    BASE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}
+    FILES
+      ${CMAKE_CURRENT_SOURCE_DIR}/ut.cppm
+)
+
+target_compile_features(qlibs.ut
+  PUBLIC
+    cxx_std_23
+)
